demo-contract-tests/Cargo.toml

```
@@ -8,6 +8,6 @@ license.workspace = true
 casper-engine-test-support = { version = "7.0.0", default-features = false }
 casper-execution-engine = { version = "7.0.0", default-features = false }
 casper-contract = { version = "4.0.0", default-features = false }
```

This should use common dependency too:

```suggestion
casper-contract = { workspace = true, default-features = false }
```